cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh

```diff
@@ -17,14 +17,14 @@
 echo "Removing empty directories from etcd..."
 
 cleanup_empty_dirs () {
-  if [ "$(${ETCDCTL} ls $1)" ]; then
-    for SUBDIR in $(${ETCDCTL} ls -p $1 | grep "/$")
+  if [ "$("${ETCDCTL}" ls "${1}")" ]; then
```

Comment:
I think it's generally fine to keep using `$1` instead of `${1}`, but I have no strong preference here.

Reply:
`$1` and `${1}` are the same, but must be changed to `${1}`, because that's the only way the verified-shellcheck.sh check can pass.

Reply:
why isn't it complaining about line 26 then?

Reply:
I don't know much about shellcheck, but I did a test. I use very-shellcheck.sh to check for unmodified etcd-empty-dird-cleanup.
```
In ./cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh line 20:
  if [ "$(${ETCDCTL} ls $1)" ]; then
                        ^-- SC2086: Double quote to prevent globbing and word splitting.
In ./cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh line 21:
    for SUBDIR in $(${ETCDCTL} ls -p $1 | grep "/$")
                                     ^-- SC2086: Double quote to prevent globbing and word splitting.
In ./cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh line 23:
      cleanup_empty_dirs ${SUBDIR}
                         ^-------^ SC2086: Double quote to prevent globbing and word splitting.
In ./cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh line 27:
    ${ETCDCTL} rmdir $1
                     ^-- SC2086: Double quote to prevent globbing and word splitting.
In ./cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh line 36:
  sleep ${SLEEP_SECOND}
        ^-------------^ SC2086: Double quote to prevent globbing and word splitting.
```
The above is all the information. It is obvious that line 26 has not been checked out. I think this is the problem of shellcheck.